proto/bluzelle.proto

```diff
@@ -18,31 +18,28 @@ import "database.proto";
 import "audit.proto";
 import "pbft.proto";
 
+message bzn_envelope
+{
+    string sender = 1;
+    bytes signature = 2;
+
+    oneof payload
+    {
+        bytes pbft_request = 3;
+        bytes database_response = 4;
```

Comment:
Will a response ever arrive via this message type?

Reply:
no, but it'll be sent out via it